### ISSUE TITLE
Answers being printed without newlines

### ISSUE THREAD

**Issue by BackwardSpy** [gleitz/howdoi#186]
When I run `howdoi`, the answer is compressed into a single line.

Example in fish shell:
```shell
~ howdoi -c python json decode
import json j = json.loads('{"one" : "1", "two" : "2", "three" : "3"}') print j['two']
```

Example in bash:
```shell
chris@galactica:~$ howdoi -c python json decode
import json j = json.loads('{"one" : "1", "two" : "2", "three" : "3"}') print j['two']
```

I have tried add/removing various flags to see if that makes a difference, but the behaviour stays consistent:

```shell
~ howdoi python json decode
import json j = json.loads('{"one" : "1", "two" : "2", "three" : "3"}') print j['two']
~ howdoi -a python json decode
★  Answer from https://stackoverflow.com/questions/7771011/parse-json-in-python ★
Very simple:
import json j = json.loads('{"one" : "1", "two" : "2", "three" : "3"}') print j['two']
```

It is not just this specific query that acts this way, every question I enter behaves like this.

It seems that all the `<pre>` blocks in each answer are being compressed into a single line. In the case where an answer contains multiple `<pre>` blocks, they each appear on separate lines.

I think this is quite a recent development, but I can't pin exactly when it started occurring. Let me know if there is any other useful information I can provide.

**Comment by WindSoilder** [comment]
Sadly found that howdoi *can print newlines* in the code in my environment.  

Would you please tell me that which python version you are running?  I can't reproduce it in `python3`  : (

**Comment by mulderns** [comment]
On my osx, howdoi 1.1.12, python 2.10, python 3.6.4 both show code blocks as single line.

**Comment by WindSoilder** [comment]
After I re-install the `howdoi`, I can reproduce this issue too.

I think this is because the dependency `pyquery` lib have publish a new version `1.4.0`.
Because when I try to install `pyquery` with previous version `1.3.0`, *howdoi* can show code correctly.

**Comment by BackwardSpy** [comment]
I just manually installed `pyquery==1.3.0` in my howdoi venv and it fixed the issue for me too.

**Comment by gleitz** [comment]
This is fixed in #187. Will release a new version shortly
